app/src/main/java/org/hackillinois/android/view/project/ProjectAdapter.kt

```diff
@@ -16,19 +17,19 @@ class ProjectAdapter internal constructor(private val mentorList: List<ProjectMo
     inner class ViewHolder(parent: View) : RecyclerView.ViewHolder(parent)
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
-        val view = LayoutInflater.from(parent.context).inflate(R.layout.mentor_list_item, parent, false)
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.project_list_item, parent, false)
         val viewHolder = ViewHolder(view)
         context = parent.context
         return viewHolder
     }
 
-    override fun onBindViewHolder(holder: ProjectAdapter.ViewHolder, position: Int) {
-        val mentor = mentorList[position]
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val project = mentorList[position]
 
         holder.itemView.apply {
-            mentorName.text = mentor.name
-            mentorNumber.text = "#${mentor.number}"
-            mentorLocation.text = mentor.location
+            mentorName.text = getMentorsString(project)
```

Comment:
Once you've changed the above function, you can just call `project.getMentorsString()`